nodes/ElseNode.java

```diff
@@ -0,0 +1,89 @@
+package nodes;
+
+import exceptions.SyntaxException;
+import provided.JottTree;
+import provided.Token;
+import provided.TokenType;
+
+import java.util.ArrayList;
+
+public class ElseNode implements JottTree {
+
+    private BodyNode bodyNode;
+
+    public ElseNode (BodyNode bodyNode) {
+        this.bodyNode = bodyNode;
+    }
+
+    @Override
+    public String convertToJott() {
+        if (this.bodyNode == null) {
+            return "";
+        } else {
+            StringBuilder str = new StringBuilder();
+            str.append("else{");
+            str.append(this.bodyNode.convertToJott());
+            str.append("}");
+            return str.toString();
+        }
+    }
+
+    @Override
+    public String convertToJava(String className) {
+        return null;
+    }
+
+    @Override
+    public String convertToC() {
+        return null;
+    }
+
+    @Override
+    public String convertToPython() {
+        return null;
+    }
+
+    @Override
+    public boolean validateTree() {
+        return false;
+    }
+
+    static public ElseNode parseElseNode(ArrayList<Token> tokens) {
+
+        // if token == ID_KEYWORD && is token == "ELSE"
+        if (tokens.get(0).getTokenType() == TokenType.ID_KEYWORD && tokens.get(0).getToken().equalsIgnoreCase("else")) {
```

Comment:
If you're already going to be checking the token string, is checking the type necessary?

Reply:
type check has been removed. i'll do the same for the while and else if nodes as well in my next commit